python/pytest_plugins/pytest_operator_groups/pytest_operator_groups/plugin.py

```diff
@@ -60,7 +60,9 @@ class Group:
     groups: set[Group] = set()
     for function in items:
         if (group_number := _get_group_number(function)) is None:
-            continue
+            raise Exception(
```

Comment:
What if we add a fallback value + a deprecation warning instead?
So it allow a more gentle adoption.

Reply:
My point was not to skip the test silently, but to fallback to something, let's say `group 1` with the addition of a loud warning.

Reply:
Can't disagree